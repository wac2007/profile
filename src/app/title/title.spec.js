import React from 'react';
import { shallow } from 'enzyme';

import Title from './title';

describe('Title Component', () => {
  it('Should match title snapshot', () => {
    const titleComponent = shallow(<Title title="Hello World" icon="profile" />);
    expect(titleComponent).toMatchSnapshot();
  });
});
