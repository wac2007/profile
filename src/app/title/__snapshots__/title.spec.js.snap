// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component Should match title snapshot 1`] = `
<h2
  className="section-title"
>
  <i
    className="fa fa-profile"
  />
  Hello World
</h2>
`;
